saleor/graphql/product/sorters.py

```diff
@@ -142,7 +141,6 @@ def description(self):
         # pylint: disable=no-member
         descriptions = {
             ProductOrderField.NAME.name: "name",
-            ProductOrderField.PRICE.name: "price",
```

Comment:
I think some client apps can already use this field. Maybe it would be safer to keep it but underneath use the minimal variant price field for sorting. At the same time, we could deprecate the "minimal price" filter and say in the description that from now the "price" filter should be used. Later we'll remove it and we'll have only one way of sorting by price.